nmostesting/suites/IS1101Test.py

```diff
@@ -664,49 +664,55 @@ def test_02_02_05_01(self, test):
                     )
                     if response.status_code != 200:
                         return test.FAIL(
-                            "The request has not succeeded: {}".format(response.json())
+                            "The streamcompatibility request for sender {} status has failed: {}"
+                            .format(sender_id, response.json())
                         )
                     state = response.json()["state"]
                     if state in ["awating_essence", "no_essence"]:
                         time.sleep(3)
                     else:
                         break
             if state != "constrained":
-                return test.FAIL("inputs are unstable.")
+                return test.FAIL("Inputs are unstable")
 
             _, response = TestHelper.do_request(
                 "GET", self.node_url + "senders/" + sender_id
             )
             if response.status_code != 200:
                 return test.FAIL(
-                    "The request has not succeeded: {}".format(response.json())
+                    "The sender {} is not available in the Node API request: {}".format(sender_id, response.json())
                 )
             sender_flow_id = response.json()["flow_id"]
             if sender_flow_id is None:
-                return test.FAIL("the sender must have a flow")
+                return test.FAIL("The sender {} must have a flow".format(sender_id))
 
             _, response = TestHelper.do_request(
                 "GET", self.node_url + "flows/" + sender_flow_id
             )
             if response.status_code != 200:
                 return test.FAIL(
-                    "The request has not succeeded: {}".format(response.json())
+                    "The sender {} is not available in the Node API has an associated flow: {}"
+                    .format(sender_flow_id, response.json())
                 )
 
             if (
                 self.flow_grain_rate[sender_id] != response.json()["grain_rate"]
                 or self.flow_width[sender_id] != response.json()["frame_width"]
                 or self.flow_height[sender_id] != response.json()["frame_height"]
             ):
-                return test.FAIL("different argument")
+                return test.FAIL(
+                    "The setting NOP constraints for frame_width, frame_height\
```

Comment:
I'm really sorry, I don't understand this message?

What are the setting NOP constraints?

Did something change when it wasn't supposed to, or not change when it was?

Reply:
In the test plan NOP constaints are described as constraints matching the current flow.

Reply:
Future test tool users won't have the test plan. NOP means no-op?

Reply:
Yes it means no-op ... does not change anything because we apply constraints that are actually matching the current flow ... I agree that the message should tell what is wrong in terms that could be understood be someone not having the test plan.